lib/configuration_methods.rb

```diff
@@ -34,6 +34,22 @@ def configured_services
     end
     attr_writer :configured_services
 
+    def show_service?(service, user)
+      unless self["services.#{service}.enable"]
+        return false
+      end
+      # If service has an 'authorized' attr, return true only if it is true or equal to user username
+      if self["services.#{service}"].has_key?("authorized")
+        if self["services.#{service}.authorized"].is_a?(TrueClass)
+          return true
+        end
+        unless user && self["services.#{service}.authorized"] == user.username
+          return false
+        end
+      end
+      true
+    end
```

Comment:
Personally I prefer verbose, it seems clearer to follow the logic, but this is ok, will change.